File: Code/Client_Prototype/Client_Prototype/Classes/Quiz.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BSD_Client.Classes
{
    public class Quiz
    {
        public int q_id { get; set; }
        public string titel { get; set; }
        public List<Frage> fragen = new List<Frage>();

        public Quiz()
        {

        }

        public Quiz(int _id, String _Titel)
        {
            q_id = _id;
            titel = _Titel;
        }

        public override String ToString()
        {
            return this.titel;
        }
    }
}

File: Code/Android-Project/Tatue-Rater2/app/src/main/java/com/htl_villach/tatue_rater/Classes/Quiz.java
package com.htl_villach.tatue_rater.Classes;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by simon on 10.03.2016.
 */
public class Quiz {
    public int q_id;
    public String titel;
    public List<Frage> fragen = new ArrayList<Frage>();

    public Quiz() {
    }

    public Quiz(int q_id, String titel) {
        this.q_id = q_id;
        this.titel = titel;
    }

    public void addFrage(Frage _f) {
        fragen.add(_f);
    }

    @Override
    public String toString() {
        return titel;
    }
}
